### DISCUSSION TITLE
4k mode

### DISCUSSION THREAD

**Discussion by Mushi4k** [bemusic/bemuse#682] [Ideas]
Could there be a 4k mode for this game? I feel like this would help the game grow as there are many 4k players

**Comment by KenZync** [comment]
bruh

**Comment by Mayapixi** [comment]
agree

**Comment by idkwhattoputhere121** [comment]
I have a 4k UHD smart tv just connect your laptop or pc or whatever to it and there you go, bemuse in 4k!

**Comment by Mayapixi** [comment]
good idea lol

**Comment by artur-py** [comment]
Sorry, but I don't think that would make much sense. Bemuse is a BMS player, a.k.a. a simulator (to some extent) of the rhythm game Beatmania. OG Beatmania is a 5k game (+ scratch lane) and it's sequel, IIDX, is a 7k game (+ scratch lane), hence Bemuse having both 5k(+1) and 7k(+1) charts. I'm sorry, but that's how things are. ¯\_(ツ)_/¯

**Reply by ibmoth** [reply]
Agreed
